server/datastore/mysql/mysql.go

```diff
@@ -1030,6 +1030,17 @@ func (ds *Datastore) whereOmitIDs(colName string, omit []uint) string {
 	return fmt.Sprintf("%s NOT IN (%s)", colName, strings.Join(idStrs, ","))
 }
 
+func (ds *Datastore) whereFilterHostsByIdentifier(identifier, stmt string, params []interface{}) (string, []interface{}) {
+	if identifier == "" {
+		return stmt, params
+	}
+
+	stmt += " AND ? IN (h.hostname, h.osquery_host_id, h.node_key, h.uuid, h.hardware_serial)"
```

Comment:
Same comment as the other one, node_key should be removed here.

Reply:
@marko-lisica is it ok to not support `osquery host id` and `node key` here, even though it's still supported (soon to be undocumented) everywhere else?

Reply:
@mostlikelee I'm ok with not supporting `osquery_host_id` and `node_key` here.